pkg/aws/cluster/mutate_cluster.go

```diff
@@ -108,6 +116,14 @@ func (m *Mutator) MutateUpdate(request *admissionv1.AdmissionRequest) ([]mutator
 		return nil, microerror.Maskf(parsingFailedError, "unable to parse old Cluster: %v", err)
 	}
 
+	capi, err := aws.IsCAPIRelease(cluster)
```

Comment:
could we circumvent the error handling by just checking if it's capi or not to make it simple?
We could also check if if the microerror is kind "..." and than return
}

Reply:
You mean we could just ignore the error here?

Reply:
Hey 🙂 , I was thinking either we could return only the error when calling `aws.IsCAPIRelease` and check the err or just only return the bool.

Reply:
Ahhh okay got it :D Sorry for auf dem Schlauch stehen.